gml_application_schema_toolbox/gui/load_wizard_wfs.py

```diff
@@ -7,7 +7,8 @@
 # Standard library
 import os
 
-import owslib
+from owslib.wfs import WebFeatureService
+from owslib.util import ServiceException
```

Comment:
Please sort imports (see contributing guidelines and recommended git hooks):

```suggestion
from owslib.util import ServiceException
from owslib.wfs import WebFeatureService
```